tests/test_worlds.py

```diff
@@ -36,11 +36,11 @@ def setUp(self):
             # the ros controller of complete_test.wbt is started when loading the world because the robot-window is open
             'Failed to contact master at',
             'Cannot initialize the sound engine',
-            'System below the minimal requirements',
             self.crashError  # To remove once #6125 is fixed
         ]
+        # Set empty.wbt as the first world (to trigger the 'System below the minimal requirements' message)
+        self.worlds = [os.path.join(os.environ['WEBOTS_HOME'], 'resources/projects/empty.wbt')]
```

Comment:
This path is wrong and it is missing the `worlds` folder:
```suggestion
        self.worlds = [os.path.join(os.environ['WEBOTS_HOME'], 'resources/projects/worlds/empty.wbt')]
```